CHANGELOG.md

```diff
@@ -2,10 +2,18 @@
 
 ## [Unreleased]
 
+### Added
+
+- Allow to pass a Dict with field names and summary strategies to the `fields` parameter in the `Summarizer` constructor
+
 ### Changed
 
 - Pin jsonschema version to <4.18 until regresssions are fixed
 
+### Fixed
+
+- Fix the documentation rendering of the `fields` parameter in the `Summarizer` constructor
```

Comment:
Please include a link to the PR in the changelog entries:

```suggestion
- Fix the documentation rendering of the `fields` parameter in the `Summarizer` constructor ([#1195](https://github.com/stac-utils/pystac/pull/1195))
```

Reply:
I followed the Changed entry, which has no PR (and there's no PR before creating a PR) ;-)

Reply:
Yeah I just noticed that. I probably missed it in a review :grimacing: